app/build.gradle

```diff
@@ -99,11 +99,10 @@ android {
 dependencies {
     // DO NOT REMOVE BELOW COMMENTED-OUT CODE BEFORE ASKING!
     //compile 'com.github.amlcurran.showcaseview:library:5.0.0'
-    //compile 'com.android.support:appcompat-v7:22.1.1'
+    compile 'com.android.support:appcompat-v7:22.+'
```

Comment:
this can cause unexpected behaviour depending on build machine. Please specify version.